### ISSUE TITLE
[🐛 Bug]: remote_server_addr

### ISSUE THREAD

**Issue by AbeRome13** [SeleniumHQ/selenium#14792]
### What happened?

giving remote server error when using local driver instance using provided code

### How can we reproduce the issue?

```shell
from selenium import webdriver

driver = webdriver.Safari()
wait = WebDriverWait(driver, 3)
driver.implicitly_wait(3)

url = ""
driver.get(url)
```

### Relevant log output

```shell
executor = SafariRemoteConnection(
               ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: SafariRemoteConnection.__init__() missing 1 required positional argument: 'remote_server_addr'

Process finished with exit code 1
```

### Operating System

MacOS Version 15.1.1 (24B91)

### Selenium version

Python 3.12

### What are the browser(s) and version(s) where you see this issue?

Safari Version 18.1.1 (20619.2.8.11.12)

### What are the browser driver(s) and version(s) where you see this issue?

Safari

### Are you using Selenium Grid?

N/A

**Comment by VietND96** [comment]
This is fixed in https://github.com/SeleniumHQ/selenium/pull/14699 and will be released in 4.27.0

**Comment by VietND96** [comment]
Release 4.27.0 is out!
